django_x509/tests/test_ca.py

```diff
@@ -614,3 +614,10 @@ def test_generate_ca_with_passphrase(self):
         ca.full_clean()
         ca.save()
         self.assertIsInstance(ca.pkey, crypto.PKey)
+
+    def test_generate_ca_with_generalizedTime(self):
+        """ Any date after 2049, should be in generalizedTime"""
+        ca = self._create_ca(validity_end=datetime(2050, 1, 1, 0, 0, 0, 0))
+        ca.full_clean()
+        ca.save()
+        self.assertIsInstance(ca.pkey, crypto.PKey)
```

Comment:
this is what I would do:

```python
self.assertEqual(datetime_so_string(ca.validity_end),
                 ca.validity_end.strftime(generalized_time)
```

Remove `self.assertIsInstance(ca.pkey, crypto.PKey)` because it does not bring any value.

Reply:
The suggested code will even work with dates set in UTCTime. Thus, test will not explicitly ensure dates to be n GeneralizedTime format.